### PR TITLE
Collapsable Sidebar func added

### PR DESCRIPTION
**Pull request by abhishek-01k**
Figma design for the flow:
https://www.figma.com/file/PzAmXUMOmwS2ljxufwv44m/PUSH-Prototypes?node-id=22221%3A11280&mode=dev

Things left:

- [x] Add New Yield Farming logo.

### CODE REVIEW

File: src/structure/Navigation.tsx [hunk 1]
```diff
@@ -30,7 +30,9 @@ import GLOBALS from 'config/Globals';
 import navigationList from 'config/NavigationList';
 
 // Create Header
-function Navigation() {
+function Navigation({
```
Comment (HarshRajat) [inline src/structure/Navigation.tsx:33]: Changes needs to persist, see how dark mode does it

File: src/config/Globals.js [hunk 2]
```diff
@@ -101,6 +101,8 @@ export default {
   CONSTANTS: {
     HEADER_HEIGHT: 70,
     LEFT_BAR_WIDTH: 260,
+    COLLAPSABLE_LEFT_BAR_WIDTH:90,
+    COLLAPSABLE_RIGHT_BAR_WIDTH:100,
```
Comment (HarshRajat) [inline src/config/Globals.js:106]: We also need to cancel the margin of all modules when sidebar is collapsed, once changes persist, you can look at margin and adjust accordingly.

Comment (HarshRajat) [reply src/config/Globals.js:106]: Also the arrow placement should be 80% down and should overlap with sidebar and interface section in the middle vertically

**Comment by abhishek-01k** [comment]
This PR can be closed because new One is made here #1396 according to the new UI of the sidebar.

**Pull request closed without merging**